event_response_tool/README.md

```diff
@@ -0,0 +1,106 @@
+# Event Response Tool
+
+## Overview
+
+The purpose of the Event Response Tool is to provide RMS style disaster response analysis in PRIME platform. It is based on the idea that the RMS users are provided with a list of events and associated weights that are meant for constructing a weighted convolution of event distribution into a single new event. For each event, the mean, standard deviation and exposure value attributes are then multiplied by their respective weight and then aggregated into a single event.
+
+To aid this type of analysis, the tool supports the following two functionalities:
+
+**1. Creating and Updating specialized Event Response Analysis Profile**
+**2. Transforming existing LayerViews and underlying ELTs**
+
+#### Analysis Profile
+The tool creates a specialized analysis profile that has the following components:
+
+- **Event Catalog**: A dummy event catalog with one or more EventIDs starting from 1.
+
+- **Simulation**: A simulation of size 'n' where n is the weighted sum of maximum trials per event.
+
+- **Loss Filters**: A set of loss filters representing every individual event in the event catalog.
+
+- **Exchange Rates Profile**: If the default FX profile UUID is not configured in the `config/event_response_config.ini` file,
+the latest FX profile available on the server.
+
+The tool also supports updating the simulation component of an existing event response analysis profile when the RMS users receive updated event weights. The user has to be provide the UUID of the existing analysis profile to the tool in order to use this feature.
```

Comment:
What is the benefit of this over creating a new one?

Reply:
TOA currently uses this feature more when compared to creating a new profile. So it is just like an add-on feature.